test/functional/core/job_spec.lua

```diff
@@ -759,7 +753,7 @@ describe('jobs', function()
   it('cannot have both rpc and pty options', function()
     command("let g:job_opts.pty = v:true")
     command("let g:job_opts.rpc = v:true")
-    local _, err = pcall(command, "let j = jobstart(has('win32') ? ['find', '/v', ''] : ['cat', '-'], g:job_opts)")
+    local _, err = pcall(command, "let j = jobstart(['cat', '-'], g:job_opts)")
```

Comment:
Here any command should work probably anyway.

Reply:
At long as the command doesn't exit quickly. That's why `cat -` is used: to make the process long-lived.

Reply:
It is checked before the job is started (IIRC), or should be at least.